addons/product/tests/test_product_attribute_value_config.py

```diff
@@ -697,3 +697,17 @@ def test_inactive_related_product_update(self):
         product.action_unarchive()
         self.assertTrue(product.active, 'The product should be unarchived.')
         self.assertEqual(product_attribut.number_related_products, 0, 'The product attribute must not have an associated product')
+
+    def test_copy_extra_prices_of_product_attribute_values(self):
```

Comment:
Missing the exclusions part of the tests =]

Reply:
I have updated the test to check that an exclusion is indeed created :)